experiments/remhos/experiment.py

```
@@ -80,6 +79,9 @@ def compute_applications_section(self):
                 self.add_experiment_variable(pk, pv, True)
             num_resources = scaled_variables["n_nodes"]
             self.add_experiment_variable(device, num_resources, True)
+        else:
+            self.add_experiment_variable(device, 1, True)
+
```

@rfhaque Maybe this should just raise an error?
```
else:
   raise ValueError("Must specify a type of scaling ...
```

With https://github.com/LLNL/benchpark/pull/701/files#r2013047812, it is now possible for there to be no scaling in the spec. This configuration would require the user to always specify the scaling  e.g. `benchpark experiment init ... remhos+cuda+single_node`, `remhos+cuda` being no longer possible.